vnext/JSI/Shared/ChakraObjectRef.h

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "jsi/jsi.h"
+
+#ifdef CHAKRACORE
+#include "ChakraCore.h"
+#else
+#ifndef USE_EDGEMODE_JSRT
+#define USE_EDGEMODE_JSRT
+#endif
+#include <jsrt.h>
+#endif
+
+#include <exception>
+#include <string>
+
+namespace Microsoft::JSI {
+
+inline void CrashUponChakraError(JsErrorCode error) {
```

Comment:
>CrashUponChakraError [](start = 12, length = 20)

Why not make Api name VerifyJsErrorElseCrash?

Reply:
I'll rename this function to be VerifyChakraErrorElseCrash.

---
In reply to: [336152839](https://github.com/microsoft/react-native-windows/pull/3445#discussion_r336152839) [](ancestors = 336152839)